### PR TITLE
Closes #65: simplify centiloid creation

### LINKED ISSUE #65: General Issue: Simplify codes in Vignette and template
### Background Information

The derivation of the centiloid values could be simplified in Vignette and template.
Per bundfussr:
 ```
    adapet <- adapet %>%
      bind_rows(
        .,
        . %>%
          filter(PARAMCD == "SUVRBFBB" & NVMETHOD == "BERKELEY FBB SUVR PIPELINE" & REFREG == "Whole Cerebellum") %>%
          mutate(
            AVAL = compute_centiloid(
              tracer = "18F-Florbetaben",
              pipeline = "BERKELEY FBB SUVR PIPELINE",
              ref_region = "Whole Cerebellum",
              suvr = AVAL
            ),
            PARAMCD = "CLBFBB",
            PARAM = "Centiloid (CL) based on BERKELEY FBB",
            AVALU = "CL"
          ),
        . %>%
          filter(PARAMCD == "SUVRBFBP" & NVMETHOD == "BERKELEY FBP SUVR PIPELINE" & REFREG == "Whole Cerebellum"),
          mutate(
            AVAL = compute_centiloid(
              tracer = "18F-Florbetapir",
              pipeline = "BERKELEY FBP SUVR PIPELINE",
              ref_region = "Whole Cerebellum",
              suvr = AVAL
            ),
            PARAMCD = "CLBFBP",
            PARAM = "Centiloid (CL) based on BERKELEY FBP",
            AVALU = "CL"
          ),
          ...
      )
    ```

The CRIT01FL derivation could be simplified either by using `condition = AVAL < 24.1`, or by removing the `restrict_derivation()` call.

### Definition of Done

_No response_

### PR DESCRIPTION
**Pull request by jackychen651**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [X] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [X] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [X] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [X] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [X] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [X] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [X] Address any updates needed for vignettes and/or templates
- [X] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [X] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [X] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [X] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [X] Link the issue in the Development Section on the right hand side.
- [X] Address all merge conflicts and resolve appropriately
- [X] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: vignettes/adpet.Rmd [hunk 1]
```diff
@@ -191,11 +191,11 @@ For our examples, we divide the process into two main steps:
...
+The first step is to define the patient- and analysis-level variables that should be kept with `keep_vars`. In this case, we group by patient, `ADSL` variables, Analysis Date (`ADT`), Relative Analysis Day (`ADY`), and Visit (`VISIT`).
 
 **2. Convert SUVR to Centiloid**
 
-The second step involves converting the original `SUVR` parameters to Centiloid using `admiralneuro::compute_centiloid()`. The formula is based on tracer, pipeline and reference region; therefore we apply filtering conditions using `admiral::derivation_slice()`, as following:
+The second step uses `admiral::derive_extreme_records()` to add new analysis records in which the original `SUVR` values are converted into Centiloid using `admiralneuro::compute_centiloid()`. The formula is based on tracer, pipeline and reference region; therefore we apply filtering conditions using the argument `filter_add`, as following:
```
[the remark this answers is not in the document]
Comment (jwang-lilly) [reply vignettes/adpet.Rmd:198]: @jackychen651, Looks great!

File: vignettes/adpet.Rmd [hunk 2]
```diff
@@ -191,11 +191,11 @@ For our examples, we divide the process into two main steps:
 
 **1. Define the Grouping Criteria**
 
-The first step is to specify how to group the data using `admiral::slice_derivation()`. In this case, we group by patient, `ADSL` variables, Analysis Date (`ADT`), Relative Analysis Day (`ADY`), and Visit (`VISIT`). We also retain records with missing Analysis Values (`is.na(AVAL)`) by setting `keep_nas = TRUE`.
+The first step is to define the patient- and analysis-level variables that should be kept with `keep_vars`. In this case, we group by patient, `ADSL` variables, Analysis Date (`ADT`), Relative Analysis Day (`ADY`), and Visit (`VISIT`).
```
[the remark this answers is not in the document]
Comment (jwang-lilly) [reply vignettes/adpet.Rmd:194]: @jackychen651, good thinking. I agree.

File: vignettes/adpet.Rmd [hunk 3]
```diff
@@ -205,88 +205,85 @@ c.  Based on the (`SUVRAFBB`) tracer, the (`AVID FBB SUVR PIPELINE`) method, and
...
-The new parameter values which include `AVAL`, `PARAMCD`, `PARAM`, and `AVALU`, are assigned within the `args` argument of this function.
+The new parameter values which include `AVAL`, `PARAMCD`, `PARAM`, and `AVALU`, are assigned within the `set_values_to` argument of this function.
+
+In this step, centiloid values are derived independently for each row that meets the specified filtering condition, with separate derivations performed for the four defined filtering conditions.
 
 ```{r, message=FALSE, warning=FALSE}
+keep_vars <- c(
+  get_admiral_option("subject_keys"),
+  adsl_vars,
+  exprs(ADT, ADY, VISIT)
```
[the remark this answers is not in the document]
Comment (jwang-lilly) [reply vignettes/adpet.Rmd:216]: get_admiral_options(), adsl_vars and exprs() all return list. Ignore Copilot.

**Comment by Fanny-Gautier** [comment]
My apologies for the delayed review, I was on vacation last week.
I don’t find `derive_extreme_records()` very easy to use; to me, it actually feels **more complicated** than other `{admiral}` or basic functions.
Could you please clarify the motivation for replacing the [first](https://github.com/pharmaverse/admiralneuro/issues/65#issuecomment-3213342971) suggestion from Stefan (`bind_rows()`) with this function?
